contracts/utils/Bootstrap.sol

```diff
@@ -121,4 +121,10 @@ contract Bootstrap is ModuleManager {
     function getInitNexusWithSingleValidatorCalldata(BootstrapConfig calldata validator) external view returns (bytes memory init) {
         init = abi.encode(address(this), abi.encodeCall(this.initNexusWithSingleValidator, (IModule(validator.module), validator.data)));
     }
+
+    /// @dev EIP712 domain name and version.
+    function _domainNameAndVersion() internal pure override returns (string memory name, string memory version) {
```

Comment:
this is for completeness?

Reply:
shouldn't be public/external?

Reply:
it is for completeness, yes.

no, it shouldn't as it is used in other public method